libsrc/_DEVELOPMENT/Winmake.bat

```diff
@@ -37,18 +37,20 @@ for %%t in (%targets%) do (
    if not "!temp!" == "%alltargets%" (
 
       set cpu=
+
+      if "%%t" == "scz180" (
```

Comment:
Flattened out the decision structure. It was getting unwieldy.
But, can't test this without a Windows build.